sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSonarRulesDefinitionTest.java

```diff
@@ -36,13 +36,13 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 class CSharpSonarRulesDefinitionTest {
-  private static final String SECURITY_HOTSPOT_RULE_KEY = "S5766";
-  private static final String VULNERABILITY_RULE_KEY = "S4426";
+  private static final String SECURITY_HOTSPOT_RULE_KEY = "S4502";
+  private static final String VULNERABILITY_RULE_KEY = "S2053";
   private static final String NO_TAGS_RULE_KEY = "S1048";
   private static final String SINGLE_PARAM_RULE_KEY = "S1200";
   private static final String MULTI_PARAM_RULE_KEY = "S110";
 
-  private static final SonarRuntime SONAR_RUNTIME = SonarRuntimeImpl.forSonarQube(Version.create(9, 9), SonarQubeSide.SCANNER, SonarEdition.COMMUNITY);
+  private static final SonarRuntime SONAR_RUNTIME = SonarRuntimeImpl.forSonarQube(Version.create(10, 10), SonarQubeSide.SCANNER, SonarEdition.COMMUNITY);
```

Comment:
I know this is cleaner - but the STIG are being raised only from SQ 10.7+ while OWASP from 9.9+ so I'd keep this tests seperate.

Reply:
`SONAR_RUNTIME` is used for all tests in this file. Only a few of them are related to security standards. I don't think using multiple runtime objects is worthwhile unless there are some breaking changes.

Reply:
Ok let's keep on single runtime - however I think we should split these tests to be on UT per standard.